rules.json

```
@@ -2,16 +2,167 @@
+    "• You are NOT ALLOWED to use inappropriate language that includes racism, sexism, or anything that is extremely offensive to specific groups of people. This can be in your status message, profile picture, all of the text chats, as well as all of the public voice channels.",
+    "• People are allowed to use PG-13 Language in any public voice channels as long as (1) consist of 8 or less people and (2) everyone in that voice lobby is okay with it.",
+    "• Do NOT try to bypass any bot filters.",
+    "• Do NOT spam in any text channel.",
+    "• Do NOT post inappropriate pictures if ever given the permission to post links/pictures.",
+    "Respect",
+    "• Respect and treat everyone EQUALLY. This means to treat and respect both members and the staff team how you would like to be treated.",
+    "• Maturity must be used at ALL TIMES, even when a Staff Member is not in the channel.",
+    "• Obey all staff members. If you feel like you are being treated unfairly by a staff member, please fill out a Staff Report on our forums. Do not complain about it in public.",
+    "• Do not impersonate ANY other members on the Discord Server. This also applies to any known YouTubers/Celebrities. This falls under “Intentionally Deceiving,” and you can be punished for it.",
+    "• Do NOT tag (using @ to ping or send a notification to a specific member) any staff members unless you have a valid reason to do so. Also, do not tag any specific users repeatedly.",
+    "• It is under your discretion to share your own personal information with other members such as race, sex, sexual orientation, financial status, full names, addresses, IP’s, appearances, and voices. However, do NOT share others’ personal information, unless given permission.",
+    "• For the sake of everyone, it is only fair and respectful that TXDPSRP is an ENGLISH only network. Please refrain from speaking any other languages than English in any text/voice channels.",
+    "Advertising",
+    "• Do NOT have a social media link, IP, or another FiveM server in your discord status message.",
+    "• Do NOT advertise another server in our text/voice channels. You may advertise another server in Private Direct Messages, but ONLY IF THEY ASK FOR IT. Failure to follow this rule will result in a permanent ban.",
+    "• Similarly, do not advertise your YouTube channel or Social Media to any users unless they ASK FOR IT.",
+    "Notes",
+    "Anything you say to users through private messages on any application can be punishable on the Discord Server as well. That includes harassment, unnecessary swearing, sending random links, etc. Message a staff member with solid evidence, such as screenshots, videos, etc, if you feel someone should be punished for something they said through Private Messages.",
+
+    "We, The TXDPSRP Network also strictly follow the Discord Terms of Service (TOS) and their Guidelines! If you are found breaking either of these you will be BANNED without notice and/or explanation",
```

Remove the comma on the last line of every list, and add missing ``]``
```suggestion
    "We, The TXDPSRP Network also strictly follow the Discord Terms of Service (TOS) and their Guidelines! If you are found breaking either of these you will be BANNED without notice and/or explanation"
 ]
```